fleetshard/config/config.go

```
@@ -35,7 +35,6 @@ type Config struct {
 	ServiceAccountTokenFile string        `env:"FLEET_MANAGER_TOKEN_FILE"`
 	CreateAuthProvider      bool          `env:"CREATE_AUTH_PROVIDER" envDefault:"false"`
 	MetricsAddress          string        `env:"FLEETSHARD_METRICS_ADDRESS" envDefault:":8080"`
-	EgressProxyImage        string        `env:"EGRESS_PROXY_IMAGE"`
```

Will deleting this remove the ENV flag and break if we try to set it? Or will the unknown flag be silently ignored?

---

I think the order in which these are loaded is in reverse.

`.Values.fleetshardSync.egressProxy.image` in the fleetshard-sync template creates the env var. The value in the env var is then loaded here.

They were both removed, but the helm value is still set (for now) in the gitops config. I think helm will silently ignore that value.

---

ACK; sounds good.